src/lib/bindRaf.js

```diff
@@ -0,0 +1,22 @@
+export const bindRaf = (fn) => {
+	let isRunning = null;
+	let that = null;
+	let args = null;
+
+	const run = () => {
+		isRunning = false;
+		fn.apply(that, args);
+	};
+
+	return () => {
+		that = this;
+		args = arguments;
```

Comment:
not using these variables, so could probably make this simpler.

Reply:
Will try to get around to fixin that tomorrow, sorry I tried a few solution before I was happy, that’s the left over mess

Reply:
@dazld Just read over the code. both variables are used.

1. self is used so that on callback the context of the function we are wrapping has the same context as the place we bound it (I think)
2. args is used to grab the arguments that were passed in upon creation of the callback.

I have made another commit to rename that to the more standard self name and removed debounce as it's unused.

The only improvement this could use would be a cancellation token, but I think that can wait as the not binding to null fix previously is good enough.

Any more issues?

Reply:
I should learn to read, sorry @falconmick - no, this all looks ok.